LabelGenerator.py

```diff
@@ -632,6 +650,12 @@ def main() -> None:
         [9100000000,   9200000000,   3300000000],
     ]
 
+    # ############################################################################
+    # Alternatively, The E12, E24, and E24_ALL constants can be passed to the 
+    # generate_decades function to create typical resistor value sets.
```

Comment:
This documentation is confusing. Id prefer something like "Alternatively, a set of common resistor values can be generated by the generate_decades function."

Reply:
Updated with suggested text.